daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -102,16 +104,22 @@ message ValName {
   // Module where the value is defined
   ModuleRef module = 1;
 
-  // value name.
+  // *Must be a non-empty list of a valid identifiers unless `interned_id` is set.*
   repeated string name = 2;
```

Comment:
```suggestion
   repeated string name = 2;  // Deprecated in version 1.dev
```
same as above.

Reply:
There is the option of supporting both schemes. Not sure it'd be sensible, but I'd like to at least discuss that. Right now the scala encoder doesn't intern for example.